migaloo/chain.json

```diff
@@ -212,6 +220,10 @@
       }
     ],
     "grpc": [
+      {
+        "address": "migaloo-grpc.polkachu.com:20790",
+        "provider": "Polkachu"
```

Comment:
Why did you reorder this?

Reply:
Only because I added Polkachu as the top entry in the other two areas and wanted to be consistent. 
Tools which use this registry to get these URLS tend to only consider the first one in the list.